.babelrc

```diff
@@ -3,7 +3,7 @@
     [
       "env",
       {
-      	"targets": { "node": "12.x" }
+      	"targets": { "node": "12.22.9" }
```

Comment:
**Question**: why do you force a specific version? in the package.json file, you don't use a specific version.